src/lib/pages/backoffice/Patron/PatronDetails/ItemsSearch/state/types.js

```diff
@@ -0,0 +1,6 @@
+//separated into different file because of a circular dependency caused otherwise
+export const IS_LOADING = 'itemsSearchBarcode/IS_LOADING';
```

Comment:
In the past, we have removed all the `types.js` and moved the consts inside `action.js` to avoid yet another file, just for that.
I would keep as it is for consistency, but this is very very minor.